src/librustc_front/hir.rs

```diff
@@ -1254,7 +1269,6 @@ pub enum Item_ {
 
     // Default trait implementations
     ///
-    // `impl Trait for .. {}`
```

Comment:
We lose this comment, I imagine it is because the empty line comment is a doc comment, it should probably lose a `/`, then hopefully rustfmt won't screw this up. Needs a fixup.